pkg/properties/propertiesDelete.go

```diff
@@ -22,35 +24,52 @@ func DeleteProperty(
 	apiServerUrl string,
 	console utils.Console,
 ) error {
-
-	err := deleteCpsProperty(namespace, name, apiServerUrl, console)
+	var err error
+	err = validateInputsAreNotEmpty(namespace, name)
+	if err == nil {
+		err = deleteCpsProperty(namespace, name, apiServerUrl, console)
+	}
 	if err == nil {
 		console.WriteString("Successfully deleted '" + name + "' in namespace '" + namespace + "'")
-	} else {
-		console.WriteString(err.Error())
 	}
 
 	return err
 }
 
+func validateInputsAreNotEmpty(namespace string, name string) error {
+	var err error
+	if len(strings.TrimSpace(namespace)) == 0 {
+		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_MISSING_NAMESPACE_FLAG, namespace)
+	}
+	if len(strings.TrimSpace(name)) == 0 {
+		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_MISSING_NAME_FLAG, name)
+	}
+	return err
+}
+
```

Comment:
You can use cobra's `MarkFlagRequired` for both `name` and `namespace`, then you won't need this function as cobra will error out if a required flag isn't provided.

Reply:
function has been added because cobra can't make them mandatory even though the flags are there because they are part of the parent flag and making them mandatory will break `properties get`